tests/acceptance/03-CheckForWarningsAndNoticesCept.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @package     redCORE
+ * @subpackage  Cept
+ * @copyright   Copyright (C) 2008 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+// Load the Step Object Page
+$I = new \AcceptanceTester($scenario);
+$I->wantToTest(' that there are no Warnings or Notices in redCORE');
+$I->doAdministratorLogin();
+$I->wantTo('Activate redCORE system plugin features');
+$I->amOnPage('administrator/index.php?option=com_plugins');
+$I->fillField(['id' => 'filter_search'], 'redcore - system plugin');
+$I->click(['xpath' => "//div[@id='filter-bar']/div[2]/button"]); // search button
+$I->click(['link' => 'redCORE - System plugin']);
+$I->waitForText('Plugin Manager: redCORE - System plugin', 5, ['css' => 'h1']);
+$I->click(['link' => 'Translation options']);
+$I->selectOptionInChosen('Enable translations', 'Yes');
+$I->click(['link' => 'Webservice options']);
+$I->selectOptionInChosen('Enable webservices', 'Yes');
+$I->click(['link' => 'OAuth2 Server options']);
```

Comment:
something seems to be wrong here:

![screen shot 2015-05-04 at 17 35 11](https://cloud.githubusercontent.com/assets/1375475/7456230/2c46e140-f283-11e4-9497-6be76a186ee4.png)

Anyway we are not yet testing the OAuth2. Feel free to remove these lines from the test for the moment.